theano/gpuarray/blas.py

```diff
@@ -495,12 +498,13 @@ def pad(self):
         return (0, 0)
 
     def __str__(self):
-        return '%s{%s, %s, %s, %s}' % (
+        return '%s{%s, %s, %s, %s, %s}' % (
             self.__class__.__name__,
             self.border_mode,
             str(self.subsample),
             str(self.filter_dilation),
-            str(self.num_groups))
+            str(self.num_groups),
+            str(self.unshared))
 
     def __setstate__(self, d):
```

Comment:
The number of flops remains the same since we're just multiplying different weights. I had initially raised a NotImplemented error before @gvtulder pointed this out.